technical-reports/resolver/syntax.md

```diff
@@ -0,0 +1,37 @@
+# Syntax
+
+## &lt;token-defs&gt;
```

Comment:
The whole concept of `<token-defs>` is something I am borrowing from other specs like [CSS Color Module 4](https://www.w3.org/TR/css-color-4/#normative) where a specific reoccurring syntax has a name.

The whole notion of an array of URIs + inline tokens appears frequently throughout the syntax, and warrants a definition that may be linked back to.

**I do not care about `<token-defs>` as a name!** As long as we give _whatever this is_ a name I think that’s sufficient.